app/views/renewal/HowCashPaymentsReceivedView.scala.html

```diff
@@ -55,14 +55,19 @@
 
     @subtitle("summary.renewal")
 
-    @heading("renewal.cash.payments.received.title")
-
     <p class="govuk-body">@messages("renewal.cash.payments.received.text")</p>
 
     @formHelper(action = controllers.renewal.routes.HowCashPaymentsReceivedController.post(edit)) {
 
         @inputCheckbox(
             Checkboxes(
+                    fieldset = Some(Fieldset(
+                        legend = Some(Legend(
+                        content = Text(messages("renewal.cash.payments.received.title")),
```

Comment:
The hint text has moved to the top of the page but should be below the heading.

<img width="1792" alt="Screenshot 2024-08-09 at 10 05 48" src="https://github.com/user-attachments/assets/acbd843a-6763-4896-b357-c38e9194d69f">

<img width="1792" alt="Screenshot 2024-08-09 at 10 09 33" src="https://github.com/user-attachments/assets/84711dcc-4c09-436f-a695-d757ea914c82">